### ISSUE TITLE
Font size is wrong

### ISSUE THREAD

**Issue by bendem** [initialcommit-com/git-sim#73]
I don't know how the images are rendered, but it looks like the fonts are not specified and it picks up default from somewhere. I have large fonts enabled anywhere I can for reasons. This is the output generated right after install with no customization applied yet:

status:
![git-sim-status_03-15-23_14-41-10](https://user-images.githubusercontent.com/2681677/225329487-c9f3c9a0-1bed-46ab-a312-c66eeabef46c.png)
merge:
![git-sim-merge_03-15-23_14-52-10](https://user-images.githubusercontent.com/2681677/225329492-9dde93e8-c111-4941-b3da-d2f7adfd53eb.png)

System infos:

```console
> cat /etc/system-release
Rocky Linux release 8.7 (Green Obsidian)

> uname -r
5.15.90.1-microsoft-standard-WSL2

> dnf list --installed ffmpeg cairo pango | cat
Installed Packages
cairo.x86_64                1.15.12-6.el8                @appstream
ffmpeg.x86_64               4.2.8-1.el8                  @rpmfusion-free-updates
pango.x86_64                1.42.4-8.el8                 @appstream

> pipx list
venvs are in /home/demarteaub/.local/pipx/venvs
apps are exposed on your $PATH at /home/demarteaub/.local/bin
   package git-sim 0.2.7, installed using Python 3.9.13
    - git-dummy
    - git-sim
```

**Comment by bendem** [comment]
bisect:
```
3533002e3e726fa991038aca347c51d5f208b8c6 is the first bad commit
commit 3533002e3e726fa991038aca347c51d5f208b8c6
Author: Jacob Stopak <jacob@initialcommit.io>
Date:   Tue Feb 28 16:38:57 2023 -0800

    Refactor all subcommands to accomodate new diverse branching system

    Signed-off-by: Jacob Stopak <jacob@initialcommit.io>

 git_sim/add.py                  |   3 +-
 git_sim/branch.py               |   2 +-
 git_sim/cherrypick.py           |   4 +-
 git_sim/commit.py               |   3 +-
 git_sim/git_sim_base_command.py | 146 ++++++++++++++++++++++++------------
 git_sim/log.py                  | 162 ----------------------------------------
 git_sim/merge.py                |   7 +-
 git_sim/rebase.py               |   7 +-
 git_sim/reset.py                |   9 ++-
 git_sim/restore.py              |   3 +-
 git_sim/revert.py               |   8 +-
 git_sim/stash.py                |   3 +-
 git_sim/status.py               |   3 +-
 git_sim/tag.py                  |   2 +-
 14 files changed, 127 insertions(+), 235 deletions(-)
```

**Comment by initialcommit-io** [comment]
@bendem Thanks for reporting this.

I believe this is a regression issue that only happens in repos with less than 5 commits.  I had originally fixed this but then overhauled the commit-parsing method, so I think I need to fix it again.

If you test on a repo with more than 5 commits I don't think you'll have this issue.  I'll let you know when it's fixed.  Let me know if any other questions about it.

**Comment by initialcommit-io** [comment]
Just released a fix for this!  Please use git-sim version 0.2.9 to take advantage of the fixes.  You can upgrade git-sim with: `pip install git-sim --upgrade`.

I'll close this for now since it's implemented, but please let me know if you have any questions or run into any issues!

**Comment by bendem** [comment]
Perfection! <3
